src/app/core/learning-object-module/submissions/submissions.routes.ts

```diff
@@ -11,16 +12,23 @@ export const SUBMISSION_ROUTES = {
             userId
         )}/learning-objects/${encodeURIComponent(learningObjectId)}/submissions`;
     },
+    CHECK_FIRST_SUBMISSION(params: {
```

Comment:
If you don't mind, could you do a bloc comment like on 24-29 that just gives a quick overview of what this route is doing

Reply:
sure thing